src/clj_gatling/pipeline.clj

```diff
@@ -56,15 +56,30 @@
     (assoc m k v)
     m))
 
+(defn max-users
+  "Determines the maximum number of concurrent requests that can be running,
+  given the rate at which they are created and time out. Rounds up"
+  [rate timeout-in-ms]
+  (-> timeout-in-ms
+      (max 60000) ;; Default to a minimum of 60s, to work around other timeouts that seem to cause issues
```

Comment:
This line could possibly be deleted. I wasn't sure whether the oddities I was seeing were due to some hidden timeout in httpkit that wasn't covered by `:timeout` in the options for it, or Google being sneaky about how they rate-limited me.

Reply:
Httpkit client has three options: `:timeout`, `:connect-timeout` and `:read-timeout`. However, if only `:timeout` is specified that should be used: https://github.com/http-kit/http-kit/blob/master/src/org/httpkit/client.clj#L227

Reply:
Done!